### PR TITLE
[ko] tutorials/distribute/save_and_load

### PR DESCRIPTION
**Pull request by lamberta**
Migrate https://github.com/tensorflow/docs/pull/1201 by @Kimminjae748
CLA signed in that PR

### CODE REVIEW

**Comment by jaketae** [comment]
@Kimminjae748 Thank you for the submission. Please take a look at the comments available at the original (now closed) PR, and we'll be able to merge this soon!

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 1]
```diff
@@ -0,0 +1,571 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "A0lG6qgThxAS"
+      },
+      "source": [
+        "## 개요\n",
+        "\n",
+        "훈련 도중 모델을 저장하고 로드하는 것은 흔히 일어나는 일입니다. 케라스 모델을 저장하고 로딩하기 위한 API에는 2가지가 있는데: high-level API와 low-level API가 해당됩니다. 이 튜토리얼은 `tf.distribute.Strategy`를 사용할 때 어떻게 SavedModel APIs를 사용할 수 있는지 보여줍니다. SavedModel와 직렬화에 관한 일반적인 내용을 학습하려면, [saved model guide](../../guide/saved_model.ipynb)와 [Keras model serialization guide](../../guide/keras/save_and_serialize.ipynb)를 읽어보는 것을 권장합니다. 간단한 예로 시작해보겠습니다:"
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "훈련 도중 모델을 저장하고 불러오는 것은 흔히 일어나는 일입니다. 케라스 모델을 저장하고 불러오기 위한 API에는 high-level API와 low-level API, 두 가지가 있습니다. 이 튜토리얼은 `tf.distribute.Strategy`를 사용할 때 어떻게 SavedModel APIs를 사용할 수 있는지 보여줍니다. SavedModel와 직렬화에 관한 일반적인 내용을 학습하려면, [saved model guide](../../guide/saved_model.ipynb)와 [Keras model serialization guide](../../guide/keras/save_and_serialize.ipynb)를 읽어보는 것을 권장합니다. 간단한 예로 시작해보겠습니다:"
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 2]
```diff
@@ -0,0 +1,571 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "L01wjgvRizHS"
+      },
+      "source": [
+        "## 모델 저장하고 로딩하기\n",
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "## 모델 저장하고 불러오기\n",
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 3]
```diff
@@ -0,0 +1,571 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "GC6GQ9HDLxD6"
+      },
+      "source": [
+        "저장에 관해서, 케라스 모델을 사용하는 경우, 케라스의`model.save()` API를 사용하는 것을 권장합니다. 저장하려는 모델이 케라스 모델이 아닌 경우, 더 낮은 단계의 API를 선택해야 합니다.\n",
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "저장에 관해서, 케라스 모델을 사용하는 경우, 케라스의 `model.save()` API를 사용하는 것을 권장합니다. 저장하려는 모델이 케라스 모델이 아닌 경우, 더 낮은 단계의 API를 선택해야 합니다.\n",
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 4]
```diff
@@ -0,0 +1,571 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "Tzog2ti7YYgy"
+      },
+      "source": [
+        "특별한 경우는 잘 정의되지 않은 입력을 갖는 케라스 모델을 갖고 있는 경우입니다. 예를 들어, 순차 모델은 입력 형태(`Sequential([Dense(3), ...]`) 없이 만들 수 있습니다. 하위 분류된 모델들 또한 초기화 후에 잘 정의된 입력을 갖고 있지 않습니다. 이 경우 저장과 로딩에서 저수준 API를 사용해야 하며, 그렇지 않으면 오류가 발생할 수 있습니다.\n",
+        "\n",
+        "모델이 잘 정의된 입력을 갖는지 확인하려면, `model.inputs` 이 `None`인지 확인합니다. `None`이 아니라면 잘 정의된 입력입니다. 입력 형태들은 모델이 `.fit`, `.evaluate`, `.predict`에서 쓰이거나 모델을 호출(`model(inputs)`)할 때 자동으로 정의됩니다. \n",
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "모델이 잘 정의된 입력을 갖는지 확인하려면, `model.inputs` 이 `None`인지 확인합니다. `None`이 아니라면 잘 정의된 입력입니다. 입력 형태들은 모델이 `.fit`, `.evaluate`, `.predict`에서 쓰이거나 모델을 호출 (`model(inputs)`) 할 때 자동으로 정의됩니다. \n",
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 5]
```diff
@@ -0,0 +1,571 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "gYAnskzorda-"
+      },
+      "source": [
+        "모델을 복원시킨 후에는 `compile()`이 이미 저장되기 전에 컴파일 되기 때문에, `compile()`을 다시 호출하지 않고도 모델 훈련을 계속 할 수 있습니다. 그 모델은 텐서플로 표준 `SavedModel`의 프로토 타입에 저장됩니다. 더 많은 정보를 원한다면, [the guide to `saved_model` format](../../guide/saved_model.ipynb)를 참고하세요.\n",
+        "\n",
+        "`tf.distribute.strategy`의 범위를 벗어나서 `model.save()`방법만 호출하는 것은 중요합니다. 범위 안에서 호출하는 것은 지원하지 않습니다. \n",
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "`tf.distribute.strategy`의 범위를 벗어나서 `model.save()` 방법을 호출하는 것은 중요합니다. 범위 안에서 호출하는 것은 지원하지 않습니다. \n",
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 6]
```diff
@@ -0,0 +1,571 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "FITHltVKQ4eZ"
+      },
+      "source": [
+        "Note: 이 문서는 텐서플로 커뮤니티에서 번역했습니다. 커뮤니티 번역 활동의 특성상 정확한 번역과 최신 내용을 반영하기 위해 노력함에도 불구하고 [공식 영문 문서](https://github.com/tensorflow/docs/blob/master/site/en/tutorials/distribute/keras.ipynb)의 내용과 일치하지 않을 수 있습니다. 이 번역에 개선할 부분이 있다면 [tensorflow/docs](https://github.com/tensorflow/docs) 깃허브 저장소로 풀 리퀘스트를 보내주시기 바랍니다. 문서 번역이나 리뷰에 참여하려면 [docs-ko@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-ko)로 메일을 보내주시기 바랍니다."
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "Note: 이 문서는 텐서플로 커뮤니티에서 번역했습니다. 커뮤니티 번역 활동의 특성상 정확한 번역과 최신 내용을 반영하기 위해 노력함에도 불구하고 [공식 영문 문서](https://www.tensorflow.org/?hl=en)의 내용과 일치하지 않을 수 있습니다. 이 번역에 개선할 부분이 있다면 [tensorflow/docs](https://github.com/tensorflow/docs-l10n) 깃허브 저장소로 풀 리퀘스트를 보내주시기 바랍니다. 문서 번역이나 리뷰에 참여하려면 [docs-ko@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs-ko)로 메일을 보내주시기 바랍니다."
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 7]
```diff
@@ -0,0 +1,571 @@
...
+        "colab_type": "text",
+        "id": "9Ejs4QVxIdAm"
+      },
+      "source": [
+        "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/tutorials/distribute/save_and_load\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />TensorFlow.org에서 보기</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/tutorials/distribute/save_and_load.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />구글 코랩(Colab)에서 실행하기</a>\n",
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs-l10n/blob/master/site/ko/tutorials/distribute/save_and_load.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />구글 코랩(Colab)에서 실행하기</a>\n",
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 8]
```diff
@@ -0,0 +1,571 @@
...
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/en/tutorials/distribute/save_and_load.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />깃허브(GitHub)소스 보기</a>\n",
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs-l10n/blob/master/site/ko/tutorials/distribute/save_and_load.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />깃허브(GitHub)소스 보기</a>\n",
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 9]
```diff
@@ -0,0 +1,571 @@
...
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs/site/en/tutorials/distribute/save_and_load.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />노트북 다운로드 하기</a>\n",
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/docs-l10n/site/ko/tutorials/distribute/save_and_load.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />노트북 다운로드 하기</a>\n",
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 10]
```diff
@@ -0,0 +1,571 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "89xNCIO5hiCj"
+      },
+      "source": [
+        "# 분석전략을 사용한 모델 저장 및 로드"
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "# 분석전략을 사용한 모델 저장 및 불러오기"
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 11]
```diff
@@ -0,0 +1,571 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "O8xfceg4Z3H_"
+      },
+      "source": [
+        "여기에 케라스 API들을 가지고 모델을 저장하고 로드하는 예가 있습니다."
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "케라스 API들을 이용해 모델을 저장하고 불러오는 예를 소개합니다."
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 12]
```diff
@@ -0,0 +1,571 @@
...
+      "metadata": {
+        "colab_type": "text",
+        "id": "gYAnskzorda-"
+      },
+      "source": [
+        "모델을 복원시킨 후에는 `compile()`이 이미 저장되기 전에 컴파일 되기 때문에, `compile()`을 다시 호출하지 않고도 모델 훈련을 계속 할 수 있습니다. 그 모델은 텐서플로 표준 `SavedModel`의 프로토 타입에 저장됩니다. 더 많은 정보를 원한다면, [the guide to `saved_model` format](../../guide/saved_model.ipynb)를 참고하세요.\n",
+        "\n",
+        "`tf.distribute.strategy`의 범위를 벗어나서 `model.save()`방법만 호출하는 것은 중요합니다. 범위 안에서 호출하는 것은 지원하지 않습니다. \n",
+        "\n",
+        "이제 모델을 로드해 `tf.distribute.Strategy`를 사용해서 훈련시킵니다:"
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "이제 모델을 불러와서 `tf.distribute.Strategy`를 사용해 훈련시킵니다:"
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 13]
```diff
@@ -0,0 +1,571 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "q1QNRYcwuRll"
+      },
+      "source": [
+        "`tf.saved_model.load()`로 로드할 수 있습니다. 그러나 저수준 단계의 API이기 때문에 (따라서 더 넓은 사용범위를 갖습니다), 케라스 모델을 반환하지 않습니다. 대신, 추론하기 위해 사용될 수 있는 기능들을 포함한 객체를 반환합니다. 예를 들어:"
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "`tf.saved_model.load()`로 불러올 수 있습니다. 그러나 저수준 단계의 API이기 때문에 (따라서 더 넓은 사용범위를 갖습니다), 케라스 모델을 반환하지 않습니다. 대신, 추론하기 위해 사용될 수 있는 기능들을 포함한 객체를 반환합니다. 예를 들어:"
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 14]
```diff
@@ -0,0 +1,571 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "x65l7AaHUZCA"
+      },
+      "source": [
+        "로드된 객체는 각각 키와 관련된 채, 여러 기능을 포함할 수 있습니다. `\"serving_default\"`는 저장된 케라스 모델이 있는 추론 기능을 위한 기본 키입니다. 이 기능을 이용하여 추론합니다: "
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "불러와진 객체는 각각 키와 관련된 채, 여러 기능을 포함할 수 있습니다. `\"serving_default\"`는 저장된 케라스 모델이 있는 추론 기능을 위한 기본 키입니다. 이 기능을 이용하여 추론합니다: "
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 15]
```diff
@@ -0,0 +1,571 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "osB1LY8WwUJZ"
+      },
+      "source": [
+        "또한 분산방식으로 로드하고 추론할 수 있습니다:"
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "또한 분산방식으로 불러오고 추론할 수 있습니다:"
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 16]
```diff
@@ -0,0 +1,571 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "hWGSukoyw3fF"
+      },
+      "source": [
+        "복원된 기능을 호출하는 것은 단지 저장된 모델로의 정방향 패쓰입니다(예상하기에). 만약 계속해서 로드된 기능을 훈련시키고 싶다면 어떻게 하실건가요? 로드된 기능을 더 큰 모델에 내장시킬 건가요? 일반적인 방법은 이 로드된 객체를 케라스 층에 싸서(wrap) 달성하는 것입니다. 다행히도, [TF Hub](https://www.tensorflow.org/hub)는 이 목적을 위해 [hub.KerasLayer](https://github.com/tensorflow/hub/blob/master/tensorflow_hub/keras_layer.py)을 갖고 있으며, 다음과 같습니다. "
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "복원된 기능을 호출하는 것은 단지 저장된 모델로의 정방향 패쓰입니다(예상하기에). 만약 계속해서 불러온 기능을 훈련시키고 싶다면 어떻게 하실건가요? 불러온 기능을 더 큰 모델에 내장시킬 건가요? 일반적인 방법은 이 불러온 객체를 케라스 층에 싸서(wrap) 달성하는 것입니다. 다행히도, [TF Hub](https://www.tensorflow.org/hub)는 이 목적을 위해 [hub.KerasLayer](https://github.com/tensorflow/hub/blob/master/tensorflow_hub/keras_layer.py)을 갖고 있으며, 다음과 같습니다. "
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 17]
```diff
@@ -0,0 +1,571 @@
...
+        "colab_type": "code",
+        "id": "clfk3hQoyKu6"
+      },
+      "outputs": [],
+      "source": [
+        "import tensorflow_hub as hub\n",
+        "\n",
+        "def build_model(loaded):\n",
+        "  x = tf.keras.layers.Input(shape=(28, 28, 1), name='input_x')\n",
+        "  # 케라스 층에 로드된 것 포장(wrap)하기\n",
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "  # 케라스 층에 불러온 것 포장(wrap)하기\n",
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 18]
```diff
@@ -0,0 +1,571 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "Oe1z_OtSJlu2"
+      },
+      "source": [
+        "볼 수 있듯이, `hub.KerasLayer`은 `tf.saved_model.load()`로부터 로드된 결과를 또 다른 모델을 만드는데 사용될 수 있는 케라스 층으로 포장(wrap)합니다. 이것은 학습에 매우 유용합니다."
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "볼 수 있듯이, `hub.KerasLayer`은 `tf.saved_model.load()`로부터 불려온 결과를 또 다른 모델을 만드는데 사용될 수 있는 케라스 층으로 포장(wrap)합니다. 이것은 학습에 매우 유용합니다."
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 19]
```diff
@@ -0,0 +1,571 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "GC6GQ9HDLxD6"
+      },
+      "source": [
+        "저장에 관해서, 케라스 모델을 사용하는 경우, 케라스의`model.save()` API를 사용하는 것을 권장합니다. 저장하려는 모델이 케라스 모델이 아닌 경우, 더 낮은 단계의 API를 선택해야 합니다.\n",
+        "\n",
+        "로딩에 있어서, 어떤 API를 사용하느냐는 로딩 API에서 얻고자 하는 내용에 따라 결정됩니다. 케라스 모델을 가져올 수 없으면(또는 가져오고 싶지 않다면), `tf.saved_model.load()`를 사용합니다. 그 외의 경우에는, `tf.keras.models.load_model()`을 사용합니다. 케라스 모델을 저장한 경우에만 케라스 모델을 반환 받을 수 있다는 점을 유의하세요. \n",
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "모델을 불러옴에 있어서, 어떤 API를 사용하느냐는 로딩 API에서 얻고자 하는 내용에 따라 결정됩니다. 케라스 모델을 가져올 수 없으면(또는 가져오고 싶지 않다면), `tf.saved_model.load()`를 사용합니다. 그 외의 경우에는, `tf.keras.models.load_model()`을 사용합니다. 케라스 모델을 저장한 경우에만 케라스 모델을 반환 받을 수 있다는 점을 유의하세요. \n",
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 20]
```diff
@@ -0,0 +1,571 @@
...
+        "\n",
+        "API들을 목적에 따라 혼합하고 짜 맞추는 것이 가능합니다. 케라스 모델을 `model.save`와 함께 저장할 수 있고, 저수준 API인, `tf.saved_model.load`로 케라스가 아닌 모델을 로드할 수 있습니다.  "
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "API들을 목적에 따라 혼합하고 짜 맞추는 것이 가능합니다. 케라스 모델을 `model.save`와 함께 저장할 수 있고, 저수준 API인, `tf.saved_model.load`로 케라스가 아닌 모델을 불러올 수 있습니다."
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 21]
```diff
@@ -0,0 +1,571 @@
...
+      },
+      "outputs": [],
+      "source": [
+        "model = get_model()\n",
+        "\n",
+        "# 케라스의 save() API를 사용하여 모델 저장하기\n",
+        "model.save(keras_model_path) \n",
+        "\n",
+        "another_strategy = tf.distribute.MirroredStrategy()\n",
+        "# 저수준 API를 사용하여 모델 로딩하기\n",
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "# 저수준 API를 사용하여 모델 불러오기\n",
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 22]
```diff
@@ -0,0 +1,571 @@
...
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "L01wjgvRizHS"
+      },
+      "source": [
+        "## 모델 저장하고 로딩하기\n",
+        "이제 사용할 모델을 가지고 있으므로 API 저장/로딩에 대해 살펴봅시다. \n",
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "이제 사용할 모델을 가지고 있으므로 API를 이용해 모델을 저장하고 불러오는 방법에 대해 살펴봅시다. \n",
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 23]
```diff
@@ -0,0 +1,571 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "Tzog2ti7YYgy"
+      },
+      "source": [
+        "특별한 경우는 잘 정의되지 않은 입력을 갖는 케라스 모델을 갖고 있는 경우입니다. 예를 들어, 순차 모델은 입력 형태(`Sequential([Dense(3), ...]`) 없이 만들 수 있습니다. 하위 분류된 모델들 또한 초기화 후에 잘 정의된 입력을 갖고 있지 않습니다. 이 경우 저장과 로딩에서 저수준 API를 사용해야 하며, 그렇지 않으면 오류가 발생할 수 있습니다.\n",
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "특별한 경우는 잘 정의되지 않은 입력을 갖는 케라스 모델을 갖고 있는 경우입니다. 예를 들어, 순차 모델은 입력 형태(`Sequential([Dense(3), ...]`) 없이 만들 수 있습니다. 하위 분류된 모델들 또한 초기화 후에 잘 정의된 입력을 갖고 있지 않습니다. 이 경우 모델을 저장하고 불러올 시 저수준 API를 사용해야 하며, 그렇지 않으면 오류가 발생할 수 있습니다.\n",
```

File: site/ko/tutorials/distribute/save_and_load.ipynb [hunk 24]
```diff
@@ -0,0 +1,571 @@
...
+        "\n",
+        "모델이 잘 정의된 입력을 갖는지 확인하려면, `model.inputs` 이 `None`인지 확인합니다. `None`이 아니라면 잘 정의된 입력입니다. 입력 형태들은 모델이 `.fit`, `.evaluate`, `.predict`에서 쓰이거나 모델을 호출(`model(inputs)`)할 때 자동으로 정의됩니다. \n",
+        "\n",
+        "여기 예시가 있습니다:"
```
Comment (jaketae) [inline site/ko/tutorials/distribute/save_and_load.ipynb]:
```suggestion
        "예시를 살펴봅시다:"
```

**Comment by lamberta** [comment]
Suggestions added. Thanks!

**Comment by jaketae** [comment]
@lamberta LGTM, thank you!

**Pull request merged**